### PR TITLE
Feature/irc channels join

### LINKED ISSUE #53: Join a channel
When user click a new channel in the list of IRC channels, It should be add to their own channels list.

### LINKED ISSUE #56: Render and update Channels

### PR DESCRIPTION
**Pull request by luciagirasoles**
IRC channel added to User channel panel, (css activated)

Close #56 
Close #53

### CODE REVIEW

File: public/assets/js/main.js [hunk 1]
```diff
@@ -217,11 +261,15 @@ showIrcChannels = data => {
 
     data.ircChannels.forEach(element => {
       let $channel = document.createElement("li");
+      $channel.setAttribute("data-name", element);
```
Comment (liannivin) [inline public/assets/js/main.js]: Why is necessary to create this attribute?

Comment (cristianbgp) [reply public/assets/js/main.js]: That is to save the actual value of the channel's name inside an attribute of the HTML
cc: @yummta

File: public/assets/js/login.js [hunk 2]
```diff
@@ -1,6 +1,12 @@
 const $inputUsername = document.getElementById("js-input-username");
 const $buttonLogin = document.getElementById("js-button-loggin");
-const socket = new WebSocket("ws://localhost:3000");
+const socket = new WebSocket(socketUrl());
+
+function socketUrl() {
+  return location.hostname == "localhost"
+    ? "ws://localhost:3000"
+    : `ws://${location.hostname}:${location.port}`;
+}
```
Comment (yummta) [inline public/assets/js/login.js]: Please, could you explain me this function?

Comment (cristianbgp) [reply public/assets/js/login.js]: When we want other computer to connect to the websocket of our localhost, they don't have to access to their localhost they have to access to our domain.

`location.hostname` returns the domain name

**Pull request merged**